Formula/step.rb

```diff
@@ -4,13 +4,6 @@ class Step < Formula
   url "https://github.com/smallstep/cli/archive/v0.8.5.tar.gz"
   sha256 "b809c4638ffd6d0c3e1bff66d87d9fbea186783735544492c34f35fe1e9f9d79"
 
-  bottle do
```

Comment:
Please don't touch the bottle block. The ci system will take care of that itself.

Reply:
Sorry about that. I accidentally ran `brew audit` with the `--new-formula` flag out of habit and it complained about the `bottle do`. My bad. Fixed